LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 BugSplat
+Copyright (c) 2022 BugSplat
```

Comment:
I suggest leaving off the year in the future.  Instead, use the text below:

"Copyright (c) BugSplat.  All rights reserved."

Reply:
Fixed in https://github.com/BugSplat-Git/bugsplat-ng/pull/92/commits/852e92f6842550ce66cddbadf47f5ca2686f9944